packages/xo-web/src/common/xo/index.js

```diff
@@ -2017,8 +2017,9 @@ export const runBackupNgJob = ({ force, ...params }) => {
 
 export const listVmBackups = remotes => _call('backupNg.listVmBackups', { remotes: resolveIds(remotes) })
 
-export const restoreBackup = (backup, sr, startOnRestore) => {
+export const restoreBackup = (backup, generateNewMacAddresses, sr, startOnRestore) => {
```

Comment:
You can use an object for optional params.

Reply:
I dont understand. Why `generateNewMacAddresses` should be an optional param ?